portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase

```diff
@@ -113,6 +113,82 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: Legacy | Automation ID: LRAC-10257 | Test Summary: Assert total amount of views in all cards"
+	@ignore = "true"
+	@priority = "4"
```

Comment:
Hey @calvinctvo, I think you forgot to change the priority. In [Test Map](https://docs.google.com/spreadsheets/d/1Fk3jlL25H1T0D8SPS8b83Uzk7cFP8uggGmenbDXm8ro/edit#gid=1922969374&range=H32) this bug is a P5, do you think it should be a P4?